### PR TITLE
Add gcf config for feeding-america and fix a bug of GCS write error

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: gcf/custom/manifest.go [hunk 1]
```diff
@@ -193,10 +194,17 @@ func ComputeManifest(
 					manifestImport.McfProtoUrl,
 					filepath.Join("/bigstore", bucket, root, "data", im, tab, "graph.tfrecord@*.gz"),
 				)
+				manifestImport.McfUrl = append(
+					manifestImport.McfUrl,
+					filepath.Join("/bigstore", bucket, root, "data", im, tab, "graph.tfrecord@*.gz"),
```
Comment (Spaceenter) [inline gcf/custom/manifest.go:199]: mcf_url should not be graph.tfrecord
mcf_url should be something like   ..../.../nodes.mcf@N

**Review by Spaceenter — CHANGES_REQUESTED** [review]

File: gcf/custom/manifest.go (hunk 1 above)
Comment (shifucun) [reply gcf/custom/manifest.go:199]: I don't see nodes.mcf@N created in the GCS though. Will they be created if I add this?

**Review by Spaceenter — APPROVED** [review]

File: gcf/lib/common.go [hunk 2]
```diff
@@ -86,9 +86,17 @@ func WriteToGCS(ctx context.Context, path, data string) error {
 		return errors.Wrap(err, "Failed to create gcsClient")
 	}
 	w := gcsClient.Bucket(bucket).Object(object).NewWriter(ctx)
-	defer w.Close()
-	_, err = fmt.Fprint(w, data)
-	return errors.WithMessagef(err, "Failed to write data to %s/%s", bucket, object)
+	_, writeErr := fmt.Fprint(w, data)
+	closeErr := w.Close()
```
Comment (Spaceenter) [inline gcf/lib/common.go:90]: Move this to after line 94?

**Review by Spaceenter — APPROVED** [review]

File: gcf/lib/common.go (hunk 2 above)
Comment (shifucun) [reply gcf/lib/common.go:90]: maybe need to close here even though write has error?

**Pull request merged**